_posts/2025-01-09-struct-decode-intro.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: post
+title: "Structured Decoding in vLLM: a gentle introduction"
+author: "Guest Post by BentoML and Red Hat"
+---
+
+**TL/DR**:
+
+- Structured decoding allows precise control over LLM output formats
+- vLLM now supports both outlines and XGrammar backends for structured decoding
+- Recent XGrammar integration brings up to 5x improvement in tokens generations
+- Upcoming v1 release focuses on enhanced performance and better batch supports
```

Comment:
Could you be more specific about the batch support?

Reply:
I updated this, can you check if this makes sense?